Source/WebCore/layout/formattingContexts/inline/InlineLine.cpp

```diff
@@ -297,6 +297,9 @@ void Line::appendInlineBoxStart(const InlineItem& inlineItem, const RenderStyle&
     // This is really just a placeholder to mark the start of the inline box <span>.
     ++m_nonSpanningInlineLevelBoxCount;
     auto logicalLeft = lastRunLogicalRight();
+    // If startbox has an offset, it should contribute to its logical left, except if it is the first box in the line.
```

Comment:
what's a startbox (this is a start of an inline box, emphasis on inline box) ? also comment should say why we are trimming here.

Reply:
I wanted to reference `Type::InlineBoxStart` name. I will replace it by `InlineBoxStart`.